app/views/security_group/_fw_rules.haml

```diff
@@ -0,0 +1,80 @@
+%h3
+  = _('Firewall Rules')
+  %table.table.table-bordered
+    %thead
+      %tr
+        %th
+          = _('Network Protocol')
+        %th
+          = _('Host Protocol')
+        %th
+          = _('Direction')
+        %th
+          = _('Remote Security Group (name - ref)')
+        %th
+          = _('IP Range')
+        %th
+          = _('Port')
+        %th
+          = _('End Port')
+        %th
+          = _('Action')
+    %tbody
+      %tr{"ng-repeat" => "fw_rule in vm.securityGroupModel.firewall_rules",
+          "ng-form"   => "rowForm",
+          "ng-if"     => "!fw_rule.deleted"}
+        %td.table-view-pf-select
+          %select{"name"                        => "network_protocol",
```

Comment:
This is a style matter, which is accepted by rubocop. The main issue with the style aligning all parameters under the first parameter instead of using a two character indent is that it makes many lines too long which breaks Rubocop. Since there are no good way to break HAML lines, I believe the two char. indent is better. Unless you have a better solution.